scripts/ci/ci_check_links.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
We should run it in .travis.yaml and have something like that:

```
if [[ "${CI:=}" != "true" || "${TRAVIS_EVENT_TYPE:=}" == "cron" ]]; then
  rebuild_ci_slim_image_if_needed
  run_check_links
else
  echo
  echo "Skip  check links"
  echo
fi
```
This way it will always run when you run it manually and when you run a cron job.